src/store/courses/thunk.js

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
Do not update the store value without checking success response from Backend. In your case even if the backend failed to perform the task, you are still saving the data, even you are saving error data also. Please always keep data checks.